src/CaptureFile/CMakeLists.txt

```diff
@@ -19,7 +19,9 @@ target_sources(
   CaptureFile
   PRIVATE CaptureFileConstants.h
           CaptureFile.cpp
-          CaptureFileOutputStream.cpp)
+          CaptureFileOutputStream.cpp
+          FileFragmentInputStream.cpp
```

Comment:
This class feels rather generic to me, should the be inside of some other module (I have no strong opinion on this though).

Reply:
I do not expect it to be used anywhere else.

Reply:
Okay, thanks. That is fine for me.